### PR TITLE
JDF-474 Fixed qstools violation for all ejb-multi-server

### PR DESCRIPTION
**Pull request by Dantheman720**
Please let me know if there are any issues.  If not, I will continue doing the rest of them.

-Dan

### CODE REVIEW

File: ejb-multi-server/install-domain.cli [hunk 1]
```diff
@@ -16,13 +16,13 @@ batch
...
 #  *** NOTE: If you are running JBoss AS 7.1.1, comment the following command ***
-/host=master/core-service=management/security-realm=ApplicationRealm/authentication=local:remove
+#/host=master/core-service=management/security-realm=ApplicationRealm/authentication=local:remove
 
 #  *** NOTE: If you are running JBoss Enterprise Application Platform 6.0 or JBoss AS 7.1, comment the following  2 _/profile_ commands.
 #            The reason is that the invocation of unsecured methods inside of a secured application is denied otherwise.
 #  ***
-/profile=default/subsystem=ejb3:write-attribute(name=default-missing-method-permissions-deny-access, value=false)
-/profile=ha/subsystem=ejb3:write-attribute(name=default-missing-method-permissions-deny-access, value=false)
+#/profile=default/subsystem=ejb3:write-attribute(name=default-missing-method-permissions-deny-access, value=false)
```
Comment (rafabene) [inline ejb-multi-server/install-domain.cli]: Why was the lines on this file commented ?

File: ejb-multi-server/pom.xml [hunk 2]
```diff
@@ -46,18 +52,18 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!-- JBoss dependency versions -->
-        <jboss.as.plugin.version>7.3.Final</jboss.as.plugin.version>
+        <version.jboss.maven.plugin>7.3.Final</version.jboss.maven.plugin>
     <!-- Define the version of the JBoss BOMs we want to import. The 
        JBoss BOMs specify tested stacks. -->
-        <jboss.bom.version>1.0.0.Final</jboss.bom.version>
+        <!--<jboss.bom.version>1.0.0.Final</jboss.bom.version> -->
```
Comment (rafabene) [inline ejb-multi-server/pom.xml]: If this property isn't being used it should be fully removed instead of commented

**Comment by rafabene** [comment]
Otherwise my comments it looks good

File: ejb-multi-server/install-domain.cli [hunk 3]
```diff
@@ -16,13 +16,13 @@ batch
 
 # disable the local authentication for applications
 #  *** NOTE: If you are running JBoss AS 7.1.1, comment the following command ***
-/host=master/core-service=management/security-realm=ApplicationRealm/authentication=local:remove
+#/host=master/core-service=management/security-realm=ApplicationRealm/authentication=local:remove
```
Comment (sgilda) [inline ejb-multi-server/install-domain.cli]: Why was this line commented? It should not have a comment.

File: ejb-multi-server/install-domain.cli [hunk 4]
```diff
@@ -16,13 +16,13 @@ batch
...
 
 #  *** NOTE: If you are running JBoss Enterprise Application Platform 6.0 or JBoss AS 7.1, comment the following  2 _/profile_ commands.
 #            The reason is that the invocation of unsecured methods inside of a secured application is denied otherwise.
 #  ***
-/profile=default/subsystem=ejb3:write-attribute(name=default-missing-method-permissions-deny-access, value=false)
-/profile=ha/subsystem=ejb3:write-attribute(name=default-missing-method-permissions-deny-access, value=false)
+#/profile=default/subsystem=ejb3:write-attribute(name=default-missing-method-permissions-deny-access, value=false)
+#/profile=ha/subsystem=ejb3:write-attribute(name=default-missing-method-permissions-deny-access, value=false)
```
Comment (sgilda) [inline ejb-multi-server/install-domain.cli]: Same here. Remoe the comment.

File: ejb-multi-server/install-domain.cli [hunk 5]
```diff
@@ -114,7 +114,7 @@ batch
 /profile=default-web/subsystem=naming:add()
 # remove this command if a 7.1.1 server is used
 #  *** NOTE: If you are running JBoss AS 7.1.1, comment the following command ***
-/profile=default-web/subsystem=naming/service=remote-naming:add
+#/profile=default-web/subsystem=naming/service=remote-naming:add
```
Comment (sgilda) [inline ejb-multi-server/install-domain.cli]: Remove the comment.

File: ejb-multi-server/pom.xml (hunk 2 above)
Comment (sgilda) [inline ejb-multi-server/pom.xml]: The comment below also goes with this one. It's one of the properties that has a different version for EAP 6.x.

**Comment by sgilda** [comment]
Other than the above, I think it looks good. After you fix the scripts (commented lines) I want to test it before we merge though.

**Comment by sgilda** [comment]
Great job, except for a few minor issues!

**Comment by Dantheman720** [comment]
Sorry about that, I made the adjustments so that I could run it on AS 7.  That file shouldn't have been submitted.

-Dan

File: ejb-multi-server/pom.xml (hunk 2 above)
Comment (Dantheman720) [inline ejb-multi-server/pom.xml]: Am I safe to remove it?

Comment (rafabene) [inline ejb-multi-server/pom.xml]: If it's not being used, yes

**Comment by rafabene** [comment]
Replaced by #600

**Pull request closed without merging**